receiver/haproxyreceiver/metadata.yaml

```diff
@@ -22,49 +22,241 @@ resource_attributes:
   haproxy.algo:
     description: load balancing algorithm
     type: string
-
-attributes:
   proxy_name:
     description: Proxy name
     type: string
   service_name:
     description: Service name (FRONTEND for frontend, BACKEND for backend, any name for server/listener)
     type: string
 
+attributes:
+  status_code:
+    description: Status code category, 1xx, 2xx, 3xx, 4xx, 5xx or other
+    type: string
+
 metrics:
-  haproxy.connection_rate:
+  haproxy.connections.rate:
     description: Number of connections over the last elapsed second (frontend). Corresponds to HAProxy's `conn_rate` metric.
     enabled: true
     gauge:
       value_type: int
       input_type: string
     unit: "{connections}"
-  haproxy.requests:
-    description: Total number of requests on this worker process since started. Corresponds to HAProxy's `CumReq` metric.
+  haproxy.sessions.count:
+    description: Current sessions. Corresponds to HAProxy's `scur` metric.
+    enabled: true
+    gauge:
+      value_type: int
+      input_type: string
+    unit: "{sessions}"
+  haproxy.connections.total:
+    description: Cumulative number of connections (frontend). Corresponds to HAProxy's `conn_tot` metric.
```

Comment:
Is it worth referencing the HA Proxy's documentation for those who want to read further about the original / underlying metric using the `extended_documentation` field?

Reply:
I can't find an authoritative documentation for haproxy metrics :(